src/groupeddataframe/splitapplycombine.jl

```diff
@@ -451,6 +478,14 @@ wrap(x::AbstractMatrix) =
     NamedTuple{Tuple(gennames(size(x, 2)))}(Tuple(view(x, :, i) for i in 1:size(x, 2)))
 wrap(x::Any) = (x1=x,)
 
+wrap_table(x::DataFrameRow) =
```

Comment:
Instead of repeating definitions used for `wrap`, why not just have this?
```julia
wrap_table(x::DataFrameRow) = throw(...)
wrap_table(x::Any) = wrap(x)

Reply:
good point, fixed